app/code/Magento/SalesGraphQl/etc/schema.graphqls

```diff
@@ -25,13 +25,23 @@ type Customer {
         filter: CustomerOrdersFilterInput @doc(description: "Defines the filter to use for searching customer orders"),
         currentPage: Int = 1 @doc(description: "Specifies which page of results to return. The default value is 1"),
         pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once. The default value is 20"),
+        sort: CustomerOrderSortInput @doc(description: "Specifies which field to sort on, and whether to return the results in ascending or descending order.")
     ): CustomerOrders @resolver(class: "Magento\\SalesGraphQl\\Model\\Resolver\\CustomerOrders") @cache(cacheable: false)
 }
 
 input CustomerOrdersFilterInput @doc(description: "Identifies the filter to use for filtering orders.") {
     number: FilterStringTypeInput @doc(description: "Filters by order number.")
 }
 
+input CustomerOrderSortInput @doc(description: "CustomerOrderSortInput specifies the field to use for sorting search results and indicates whether the results are sorted in ascending or descending order.") {
+    sort_field: CustomerOrderSortableField! @doc(description: "Specifies the field to use for sorting")
+    sort_direction: SortEnum! @doc(description: "This enumeration indicates whether to return results in ascending or descending order")
+}
+
+enum CustomerOrderSortableField @doc(description: "Specifies the field to use for sorting") {
+    ORDER_ID @doc(description: "Sorts customer orders by ID")
```

Comment:
Can we add CREATED_AT  so that we can display the latest order as the first listed?

Reply:
Hi @fooman. Yes, we can add CREATED_AT field additionally, or instead of ORDER_ID. What should I do?